python/grass/pygrass/tests/benchmark.py

```diff
@@ -14,15 +14,14 @@
 import sys
 from jinja2 import Template
 from pathlib import Path
-
-sys.path.append(str(Path.cwd()))
-sys.path.append("%s/.." % (str(Path.cwd())))
-
 import grass.lib.gis as libgis
 import grass.lib.raster as libraster
 import grass.script as gs
 import ctypes
 
+sys.path.append(str(Path.cwd()))
+sys.path.append("%s/.." % (str(Path.cwd())))
+
```

Comment:
I assume that changing the paths was made to make the ctypes-library loading work correctly at one point in time. I'm surprised the tests seemed to pass. (If it is the only one run, or the first file run, without anything else loaded before, does it still work?)

Reply:
Is this actually running in CI? I would think it's not.

Reply:
I created this PR with intention of making updates just in case that was the case, but since all the checks passed, I left it as it is

Reply:
There's a way to know, to intentionally make a test having a failed assertion (or changing the expected value), and look if it fails. It doesn't have to be done in CI here, could be local or in your fork.

Reply:
I wasn't really convinced by any of the tests so I did a workaround. I imported the modules via functions which resolved the flake8 errors but at the same time allowed me to set path before importing modules

Reply:
Added `#noqa` in line with other similar errors for now